pkg/integrations/prometheus/promql/metricsSearchHandler.go

```diff
@@ -1236,16 +1238,28 @@ func convertPqlToMetricsQuery(searchText string, startTime, endTime uint32, myid
 	return []structs.MetricsQueryRequest{*metricQueryRequest}, pqlQuerytype, []structs.QueryArithmetic{}, nil
 }
 
-func getArithmeticOperation(op parser.ItemType) segutils.ArithmeticOperator {
+func getLogicalAndArithmeticOperation(op parser.ItemType) segutils.LogicalAndArithmeticOperator {
 	switch op {
 	case parser.ADD:
-		return segutils.Add
+		return segutils.LetAdd
 	case parser.SUB:
-		return segutils.Subtract
+		return segutils.LetSubtract
 	case parser.MUL:
-		return segutils.Multiply
+		return segutils.LetMultiply
 	case parser.DIV:
-		return segutils.Divide
+		return segutils.LetDivide
+	case parser.GTR:
+		return segutils.LetGreaterThan
+	case parser.GTE:
+		return segutils.LetGreaterThanOrEqualTo
+	case parser.LSS:
+		return segutils.LetLessThan
+	case parser.LTE:
+		return segutils.LetLessThanOrEqualTo
+	case parser.EQLC:
+		return segutils.LetEquals
+	case parser.NEQ:
+		return segutils.LetNotEquals
```

Comment:
Why are you calling these `Let...`?

Reply:
In the past, `Operation` is `ArithmeticOperator` type. Since `ArithmeticOperator` only contains `+ - * /`, so I changed  `Operation` type from `ArithmeticOperator` to `LogicAndArithmeticOperator`
```
type QueryArithmetic struct {
Operation   utils.ArithmeticOperator
}
```

Reply:
I see. I didn't realize those were already defined before this PR